src/cs50/sql.py

```diff
@@ -393,6 +393,10 @@ def teardown_appcontext(exception):
                 elif command in ["DELETE", "UPDATE"]:
                     ret = result.rowcount
 
+                # If CREATE VIEW, return True
+                elif command == "CREATE VIEW":
```

Comment:
Why special-case `CREATE VIEW` instead of just `CREATE` for, e.g., `CREATE TABLE` also?